README.md

```diff
@@ -1 +1,72 @@
-# js-libp2p-daemon-client
+libp2p-daemon client JavaScript implementation
+======
+
+> A Javascript client to interact with a standalone deployment of a libp2p host, running in its own OS process. Essentially, this client allows to communicate with other peers, interact with the DHT, participate in pubsub, etc. no matter the language they are implemented with.
+
+## Lead Maintainer
+
+[Vasco Santos](https://github.com/vasco-santos)
+
+## Table of Contents
+
+* [Specs](#specs)
+* [Install](#install)
+* [Usage](#usage)
+* [API](#api)
+* [Contribute](#contribute)
+* [License](#license)
+
+## Specs
+
+The specs for the daemon are currently housed in the go implementation. You can read them at [libp2p/go-libp2p-daemon](https://github.com/libp2p/go-libp2p-daemon/blob/master/specs/README.md)
+
+## Install
+
+`npm install libp2p-daemon-client`
+
+## Usage
```

Comment:
I think it might be helpful to add a section/subsection in the Readme to point to the go and js daemons. I think it will be valuable for people to know that this client should be able to function for both daemons. It's entirely possible this js client could get updated to support more daemon features from the go daemon, if they're released sooner.